src/components/ChatContent/ChatContent.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <transition name="fade">
+    <div id="chat-content" class="chat-content" v-show="!$store.state.minimized">
+      <chat-tabs></chat-tabs>
+      <ConnectingPopup></ConnectingPopup>
```

Comment:
Whoops
I think I will move to Pascal case; I am not a huge fan of it visually, but after what you said and what I've read I think I'd rather have the distinction between components and tags, especially since all the debug tools have these distinctions.